### ISSUE TITLE
repo decomposition

### ISSUE THREAD

**Issue by h1alexbel** [l3r8yJ/blamer#11]
I think we can avoid usage of monorepo.

@l3r8yJ WDYT?

**Comment by l3r8yJ** [comment]
@h1alexbel yes, but I want to do this solution in one place, I think we need some sort of organisation

**Comment by l3r8yJ** [comment]
@h1alexbel You can create an organisation called `Blamer', I'm not good at all this manager stuff

**Comment by h1alexbel** [comment]
@l3r8yJ hah, ok, I will create it

**Comment by h1alexbel** [comment]
@l3r8yJ `The name 'Blamer' is already taken.`

**Comment by l3r8yJ** [comment]
@h1alexbel i guess need to delete this repo
